scenarios/GitHubLovesSparkDotNet/GitHubLovesSparkDotNet.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
should we put this as part of the examples project?

Reply:
I was thinking of using this folder mostly for real-world scenarios (that leverage public datasets). What do you think? Examples folder was meant for toy examples. In the next PR (or future), I was planning on adding a whole README to each scenario and explaining how we did something sophisticated.

Reply:
but this structure leads us to maintain another solution... and updating csproj whenever there is new nuget released, etc. Why can't you do the same under example?

Reply:
Or at least just create a project that Microsoft.Spark can reference (similar to examples),
Or do something similar to benchmark such that you reference Mcirosoft.Spark project instead of nuget?

Reply:
but I think if you maintain scenario/example, people may be confused (or at least I would be)

Reply:
or maybe this scenario is for something outside of this repo.

Reply:
Ok moved to examples.